Common/util/path.cpp

```diff
@@ -173,6 +173,28 @@ String FixupSharedFilename(const String &filename)
     return fixed_name;
 }
 
+#if AGS_PLATFORM_OS_ANDROID
+    String GetPathInForeignAsset(const String &filename)
+    {
+        if(filename.IsNullOrSpace()) return filename;
```

Comment:
This is not significant, but in general it's best to use String.IsEmpty(), because it tests for empty string (uninitialized or zero length), while NullOrSpace parses whole string to see if it only contains whitespace chars.

Reply:
Ah, I mistook what the function did, I did intent for IsEmpty then, I will adjust.